skipper.go

```diff
@@ -1707,7 +1717,10 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 	}
 
 	if len(o.LuaModules) > 0 {
```

Comment:
I thin I should be `if len(o.LuaModules) > 0 || len(o.LuaSources) > 0)`

Reply:
@AlexanderYastrebov  This will change the result for library users, because `len(o.LuaSources) == 0` for the default.

Reply:
I dropped the condition as we talked